src/geometries/ExtrudeGeometry.js

```diff
@@ -784,6 +784,7 @@ const WorldUVGenerator = {
 function toJSON( shapes, options, data ) {
 
 	data.shapes = [];
+	data.options = Object.assign( {}, options );
```

Comment:
Should this be moved to line 804 for readability?

Reply:
Indeed!